lib/inputs/dovetail-downloads.js

```diff
@@ -58,11 +58,12 @@ module.exports = class DovetailDownloads {
 
   async format(record) {
     const epoch = timestamp.toEpochSeconds(record.timestamp || 0);
+    const listenerSession = timestamp.toDigest(record.listenerEpisode, epoch);
     const lookups = [lookip.look(record.remoteIp), lookagent.look(record.remoteAgent)];
     const [geo, agent] = await Promise.all(lookups);
 
     return {
-      insertId: `${record.listenerEpisode}/${epoch}`,
+      insertId: `${listenerSession}/${epoch}`,
```

Comment:
This changed from episode -> session, but shouldn't matter.  Since the epoch-seconds was already in there.